django/contrib/admin/decorators.py

```diff
@@ -1,3 +1,63 @@
+def action_method(*, permissions=None, description=None):
+    """
+    Conveniently add attributes to an action method::
+
+        @admin.action_method(
+            permissions=['publish'],
+            description='Mark selected stories as published',
+        )
+        def make_published(self, request, queryset):
+            queryset.update(status='p')
+
+    This is equivalent to setting the attributes on the method directly::
```

Comment:
It's a little confusing to not signpost the mismatch between the attribute names. Especially the wording "the attributes" sounds like they should be the same. Perhaps

```suggestion
    This is equivalent to setting some attributes (with the original, longer names) on the function directly::
```

Reply:
That's nicer. And I'll change to generally prefer function throughout as methods are functions bound to an object.